src/Bicep.Core/Intermediate/Expression.cs

```diff
@@ -79,6 +79,22 @@ public override void Accept(IExpressionVisitor visitor)
         => visitor.VisitNullLiteralExpression(this);
 }
 
+public record NoneLiteralExpression(
+    SyntaxBase? SourceSyntax
+) : Expression(SourceSyntax)
+{
+    public override void Accept(IExpressionVisitor visitor)
+        => visitor.VisitNoneLiteralExpression(this);
+}
+
+public record ExtendsDeclarationExpression(
+    SyntaxBase? SourceSyntax
+) : Expression(SourceSyntax)
+{
+    public override void Accept(IExpressionVisitor visitor)
+        => visitor.VisitExtendsDeclarationExpression(this);
+}
```

Comment:
Does this need to be represented in the expression tree?

Reply:
to be honest, I'm not sure. with this commit (https://github.com/Azure/bicep/pull/13900/commits/4b89c4947a690512acc0036e8a1835a0fa086ed3) I removed it, if you think it'd be better to have it in the expression tree, I can add it back 👍